### ISSUE TITLE
nan (not a number) error

### ISSUE THREAD

**Issue by nguyenkh** [erickrf/multiffn-nli#2]
Hi,

Can you check it out. I get the error:

Reading data from snli_1.0_train.jsonl
Reading data from snli_1.0_dev.jsonl
Converting words to indices
Creating model
Starting training
0 completed epochs, 100 batches Average training batch loss: nan        Validation loss: nan    Validation accuracy: 0.338244
0 completed epochs, 200 batches Average training batch loss: nan        Validation loss: nan    Validation accuracy: 0.338244
0 completed epochs, 300 batches Average training batch loss: nan        Validation loss: nan    Validation accuracy: 0.338244
0 completed epochs, 400 batches Average training batch loss: nan        Validation loss: nan    Validation accuracy: 0.338244

**Comment by erickrf** [comment]
It was a problem with the default maximum value to clip tensor norms during training. It is fixed now.
